package cms_8120;
import "type/cmd.proto";
option java_package = "com.hikvision.finance.fms.cms.hpp.center";
option java_outer_classname = "OvertimeRecordProto";
option java_generic_services = true;

//数据请求消息
message ReqOvertimeRecord
{
    optional int32 cmd                      = 1;    // CMD_DT_OVERTIMERECORD_LIST
    //optional int32 tipflag                  = 2;    // 录像天数
}

//过期录像信息
message OvertimeRecord
{
    optional int32 id                       = 1;    // 录像ID
    optional string proId                   = 2;    // 
    optional string businessTime            = 3[default = "1970-01-01 00:00:00"];    // 
    //optional int32 tipflag                  = 4;
}

//数据响应信息
message RspOvertimeRecord
{
    optional int32 cmd                      = 1;    // CMD_DT_OVERTIMERECORD_LIST
    repeated OvertimeRecord overtimeRecord  = 2;    // 过期录像信息列表
}


service ModifyOvertimeRecordService {
	
	rpc QyeryAlarmlogID(ReqOvertimeRecord) returns (RspOvertimeRecord);			//CMD_DT_OVERTIMERECORD_LIST--获取过期录像信息
	
}
